profiles/lib/way_handlers.lua

```diff
@@ -521,7 +521,7 @@ function WayHandlers.blocked_ways(profile,way,result,data)
     local construction = way:get_value_by_key('construction')
 
     -- Of course there are negative tags to handle, too
-    if construction and construction ~= 'no' and construction ~= 'widening' then
+    if construction and construction ~= 'no' and construction ~= 'widening' and construction ~= 'minor' then
```

Comment:
What would be cool however is a whitelist in the profile table. Then you could do:

```lua
if construction and not profile.construction_whitelist[construction] then
  ...
end
```